### ISSUE TITLE
Adobe Digital Editions key

### ISSUE THREAD

**Issue by michiru7422** [apprenticeharper/DeDRM_tools#450]
My computer recently died, so I bought a new one.  But for some reason, I cannot open the books I originally opened in Aldiko using Adobe Digital Editions.  On my old computer, I could, but it has died.  I get an error when I try to do this on the new computer, saying that it cannot connect to the Aldiko server.  I have the same problem on my iPad.  I verified this by going to the URL it sent ADE to, and it is more or less broken.  

I thought to strip the DRM protection from the books so that I could read them, but it seems that the key used to do this in DeDRM tools is taken from Adobe Digital Editions, which of course does not have the key.  I have the log in information though...  

Can you please help?

**Comment by ElleKayEm** [comment]
Have you authorized Adobe Digital Editions with your Adobe ID?

**Comment by michiru7422** [comment]
Yes, it is authorized with my Adobe ID, but it refuses to open books that I know are attached to that ID.  I don't know what is going on with it...

**Comment by ElleKayEm** [comment]
Yeah, if you can't open them with ADE, then DeDRM won't be able to either.  I'd try downloading fresh copies of the books with the current installation of ADE 2.0.1 on the same machine and user as calibre/DeDRM.
